src/target.zig

```diff
@@ -207,10 +207,6 @@ pub fn supports_fpic(target: std.Target) bool {
     return target.os.tag != .windows and target.os.tag != .uefi;
 }
 
-pub fn isSingleThreaded(target: std.Target) bool {
-    return target.isWasm();
-}
-
```

Comment:
I think it would be better to keep this logic, even if it says `return false` for now. There are other targets for which it does apply, and it is currently returning `true` incorrectly for them.